.github/workflows/release.yml

```diff
@@ -317,49 +318,16 @@ jobs:
           pattern: wasm-*
           merge-multiple: true
           path: packages/@biomejs
-
       - name: Install Node.js
         uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4.4.0
         with:
           node-version: 20
           registry-url: 'https://registry.npmjs.org'
 
-      - name: Generate npm packages
-        run: node packages/@biomejs/biome/scripts/generate-packages.mjs
-
       - name: Publish npm packages as latest
-        run: for package in packages/@biomejs/*; do if [ $package != "packages/@biomejs/js-api" ]; then npm publish $package --tag latest --access public --provenance; fi; done
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: pnpm publish
 
       - name: Upload assets to the latest tag
         run: |
-          files=$(ls biome-* | tr '\n' ' ')
+          files=$(ls cli-* | tr '\n' ' ')
```

Comment:
I think `biome-` was correct, the archive file is `cli-` but the contained file is named `biome-`:

<img width="272" alt="image" src="https://github.com/user-attachments/assets/6b47de68-4cbb-4b32-a6de-92d93708d38c" />


```suggestion
          files=$(ls biome-* | tr '\n' ' ')
```